extensions/ql-vscode/src/run-queries.ts

```diff
@@ -226,6 +226,7 @@ async function convertToQlPath(filePath: string): Promise<string> {
       const fileName = path.basename(filePath);
       const fileNames = await promisify<string, string[]>(fs.readdir)(dir);
       for (const name of fileNames) {
+        // leave the locale argument empty so that the default OS locale is used.
```

Comment:
```suggestion
        // Leave the locale argument empty so that the default OS locale is used.
        // We do this because this operation works on filesystem entities, which 
        // use the os locale, regardless of the locale of the running VS Code instance.
```